orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/tasks/CreateBakeTask.groovy

```diff
@@ -136,6 +140,10 @@ class CreateBakeTask implements RetryableTask {
 
     Map requestMap = packageInfo.findTargetPackage(allowMissingPackageInstallation)
 
+    requestMap.packageArtifacts = stage.context.packageArtifactIds.collect { String artifactId ->
```

Comment:
is there any risk that `packageArtifactIds` wouldn't be present in the stage's context?  if so would suggest adding a guard here